### DISCUSSION TITLE
🚀 Share Your Experience with Zarr V3! 🚀

### DISCUSSION THREAD

**Discussion by sanketverma1704** [zarr-developers/zarr-python#2093] [Announcements]
The Zarr V3 specification is here, and we've seen the community start to implement it. Now, we want to hear from you! Whether you’ve embraced Zarr V3 in your projects or are just getting started, your feedback is invaluable.

📝 Tell Us About Your Experience: [Zarr V3 Community Feedback Survey](https://forms.gle/vjDaMMh1Sx7RY8qo7)

Help us understand how Zarr V3 is working for you and where we can improve. Your insights will help us make Zarr even better for everyone!

Thank you for being an essential part of the Zarr community!

**Comment by rabernat** [comment]
I don't understand what sort of feedback we are looking for here. V3 support in Zarr Python hasn't been released yet. How can people give feedback on it?

**Reply by sanketverma1704** [reply]
The survey is to get input from the community of users and developers on their general impression of the V3 specification, whether they have tried it out yet (there are implementations other than Zarr-Python that support V3), and what's the most exciting part of V3.

Other questions in the survey concern general community elements like social media, community meetings, and the chat platform.

I think the most important reason for this short survey is to let everyone know that we're here to listen to them if they have any questions.

**Reply by keller-mark** [reply]
This survey has a lot of unrelated but required questions that discourage filling it out. 

My feedback is that I see zarr V3 being discussed a lot as if it was production-ready today. While the "spec" might be complete and has some level of implementation already, I would argue that until V3 is supported in a major version of `zarr-python`, it is not production-ready. It should be made abundantly clear what the current state of V3 is somehow. Is the spec still evolving? Are we just waiting on implementers? Is it alpha/beta/production-ready? What are the breaking changes? What motivated / required those breaking changes? What is the migration path for those who have a little/lot of V2 data around? Should new users start with V3 or V2?  What tools are available to assist users in migration from V2 to V3?

With this it is possible that the ecosystem could fracture into V2 and V3, like python.

**Comment by carlthome** [comment]
Just lurking the project because I like the ideas of it so not read up on everything but is there a high-level list of what's changed with V3?

**Reply by d-v-b** [reply]
I'm not aware of such a document, but we should prepare one.

**Reply by sanketverma1704** [reply]
There is a small section in the V3 spec which states changes w.r.t. to V2 spec: https://zarr-specs.readthedocs.io/en/latest/v3/core/v3.0.html#comparison-with-zarr-v2.

I agree that it still needs to be fully updated because we're going to remove implicit groups soon: https://github.com/zarr-developers/zarr-specs/pull/292.
